dg-commons/pom.xml

```diff
@@ -10,20 +10,123 @@
     <modelVersion>4.0.0</modelVersion>
 
     <artifactId>dg-commons</artifactId>
+    <groupId>org.finra.datagenerator.commons</groupId>
+    <version>2.2-SNAPSHOT</version>
     <name>Data Generator Common Utils</name>
 
     <dependencies>
+        <!-- Used for base-64 encoding for HTTP basic authentication (which we need to access Jenkins). -->
+        <dependency>
+            <groupId>commons-codec</groupId>
+            <artifactId>commons-codec</artifactId>
+            <version>1.10</version>
+        </dependency>
+        <dependency>
+            <groupId>org.slf4j</groupId>
+            <artifactId>slf4j-log4j12</artifactId>
+            <version>1.7.5</version>
+        </dependency>
+        <dependency>
+            <groupId>com.jcraft</groupId>
+            <artifactId>jsch</artifactId>
+            <version>0.1.53</version>
+        </dependency>
+        <dependency>
+            <groupId>uk.com.robust-it</groupId>
+                <artifactId>cloning</artifactId>
+            <version>1.9.1</version>
+        </dependency>
+        <dependency>
+            <groupId>org.apache.commons</groupId>
+            <artifactId>commons-math3</artifactId>
+            <version>3.5</version>
+        </dependency>
+        <dependency>
+            <groupId>org.scalatest</groupId>
```

Comment:
Do we need that for unit tests ? if so, the scope should be testing. The default maven scope is compile.